src/unix/linux_like/linux/mod.rs

```diff
@@ -3019,6 +3019,19 @@ pub const ALG_SET_AEAD_AUTHSIZE: ::c_int = 5;
 pub const ALG_OP_DECRYPT: ::c_int = 0;
 pub const ALG_OP_ENCRYPT: ::c_int = 1;
 
+// include/uapi/linux/if.h
+pub const IF_OPER_UNKNOWN: u8 = 0;
+pub const IF_OPER_NOTPRESENT: u8 = 1;
+pub const IF_OPER_DOWN: u8 = 2;
+pub const IF_OPER_LOWERLAYERDOWN: u8 = 3;
+pub const IF_OPER_TESTING: u8 = 4;
+pub const IF_OPER_DORMANT: u8 = 5;
+pub const IF_OPER_UP: u8 = 6;
+
+pub const IF_LINK_MODE_DEFAULT: u8 = 0;
+pub const IF_LINK_MODE_DORMANT: u8 = 1;
+pub const IF_LINK_MODE_TESTING: u8 = 2;
```

Comment:
Should these really be `u8`? For example, `IFF_*` consts are declared as `c_int` and I'm concerned that type inconsistency confuses users.

Reply:
Updated to use c_int